app/src/main/java/com/google/droidjump/MainActivity.java

```diff
@@ -96,6 +104,37 @@ public PlayersClient getPlayersClient() {
         return playersClient;
     }
 
+    public void countTimeOfPlaying() {
+        String leaderboardId = getResources().getString(R.string.leaderboard_best_time);
+        if (savedSignedInAccount != null && LevelManager.getLastLevelIndex() != LevelManager.getLevelsLastIndex()) {
+            leaderboardsClient.loadCurrentPlayerLeaderboardScore(
+                    leaderboardId,
+                    LeaderboardVariant.TIME_SPAN_ALL_TIME,
+                    LeaderboardVariant.COLLECTION_PUBLIC).addOnSuccessListener(
+                    result -> {
```

Comment:
This is a very deep level of nesting here which makes it much harder to understand. Could something be moved in a separate method?

Reply:
Moved.